### PR TITLE
Generate graph of Sequel model associations

### PR DESCRIPTION
**Pull request by rafatower**
Story details: https://app.clubhouse.io/cartoteam/story/107021

### CODE REVIEW

File: app/models/visualization/external_source.rb [hunk 1]
```diff
@@ -5,7 +5,6 @@ module CartoDB
   module Visualization
 
     class ExternalSource < Sequel::Model
-      many_to_one :visualization
```
Comment (thedae) [inline app/models/visualization/external_source.rb:8]: Just to make sure: I guess this is removed since there's a method defined later, right?

Comment (rafatower) [reply app/models/visualization/external_source.rb:8]: yes, and that association is "shadowed" by that method, and it does not work. Documented in the commit message :) https://github.com/CartoDB/cartodb/pull/15865/commits/3f59af660bac2a84bdf4b91e8a0ba3b1dd0cc47e

**Review by thedae — APPROVED** [review]
The script looks good, this is going to help a lot :+1:

File: app/models/visualization/external_source.rb (hunk 1 above)
Comment (thedae) [reply app/models/visualization/external_source.rb:8]: You're right, my apologies!

Comment (rafatower) [reply app/models/visualization/external_source.rb:8]: Elaborating a bit further: it is using the `Visualization::Member` which belongs in "the 3rd ORM" (in-house, virtus), which is mostly ported to AR `Carto::Visualization`. Another thing in the :dart:

**Review by amiedes — APPROVED** [review]
Really interesting work 🙂 🚀

**Pull request merged**